packages/wdio-cli/src/interface.js

```diff
@@ -0,0 +1,155 @@
+import chalk from 'chalk'
+import cliSpinners from 'cli-spinners'
+import EventEmitter from 'events'
+import CLIInterface from 'wdio-interface'
+
+const clock = cliSpinners['clock']
```

Comment:
Looking down at the code I had no idea what clock was. Perhaps rename to clockSpinner?